src/main/kotlin/thatline/localup/LocalupApplication.kt

```
@@ -1,13 +1,13 @@
+package thatline.localup
+
+import org.springframework.boot.autoconfigure.SpringBootApplication
+import org.springframework.boot.context.properties.ConfigurationPropertiesScan
+import org.springframework.boot.runApplication
+
+@SpringBootApplication
+@ConfigurationPropertiesScan(basePackages = ["thatline.localup.common.property", "thatline.localup.storage.property"])
+class LocalupApplication
+
+fun main(args: Array<String>) {
+    runApplication<LocalupApplication>(*args)
```

detekt (1.23.8)

[warning] In most cases using a spread operator causes a full copy of the array to be created before calling a method. This may result in a performance penalty.

(detekt.performance.SpreadOperator)